### DISCUSSION TITLE
query Builder

### DISCUSSION THREAD

**Discussion by majchinthaka** [rappasoft/laravel-livewire-tables#455] [Q&A]
Why not use like this?

`public function query(): Builder
    {
        return DB::table('test')->select('*');
    }`

Error:
`TypeError
Return value of App\Http\Livewire\UserTable::query() must be an instance of Illuminate\Database\Eloquent\Builder, instance of Illuminate\Database\Query\Builder returned (View: C:\Users\JAN\Desktop\affiliate_admin\resources\views\test.blade.php) `

**Comment by rappasoft** [comment]
It currently only supports Eloquent.
